Incorporate anchor/elasticsearch-scripts into ES container

glasnt opened the pull request.

glasnt committed:
Incorporate anchor/elasticsearch-scripts into ES container

olorin commented on elasticsearch/Dockerfile:
```diff
@@ -27,14 +27,19 @@ RUN \
+  cd / && \
+  wget https://github.com/anchor/elasticsearch-scripts/archive/v1.0.1.tar.gz -q \
+	-O es-scripts-1.0.1.tar.gz && \
+  tar xvzf es-scripts-1.0.1.tar.gz -C /usr/local/sbin --strip 1
```
I'm not sure this will work (it won't install deps). Why not install `python2-pip` and `pip install elasticsearch-scripts` to grab everything from PyPI?

glasnt replied:
.. I am very silly and require more coffee. Rectifying.

glasnt committed:
Use pip instead of github for installation of elasticsearch-scripts

anchor/elasticsearch-scripts updated for pip<1.4 compatability (wheezy
packages pip1.1)
